aptos-move/aptos-vm/src/sharded_block_executor/test_utils.rs

```diff
@@ -188,7 +188,7 @@ pub fn sharded_block_executor_with_random_transfers<E: ExecutorClient<FakeDataSt
     let mut rng = OsRng;
     let max_accounts = 200;
     let max_txns = 1000;
-    let num_accounts = rng.gen_range(1, max_accounts);
+    let num_accounts = rng.gen_range(2, max_accounts);
```

Comment:
Why would the tests fail when we have 1 account? Is this something we need to handle in the partitioner?

Reply:
because somewhere in the txn generator we requires the transfer to be between 2 different accouts...

Reply:
i.e., [here](https://github.com/aptos-labs/aptos-core/blob/0c0aa7fe09954d9d21652e49f68a4cb05b56f09c/aptos-move/aptos-vm/src/sharded_block_executor/test_utils.rs#L206)